### PR TITLE
Mafia Component

### PR DESCRIPTION
**Pull request by LetterN**
## About The Pull Request
Title.

## Why It's Good For The Game
Ghosts can do things while waiting for roundend

## Changelog
:cl:
add: Mafia Component
/:cl:

- [x] update mapping things
- [x] do the tgui

### CODE REVIEW

File: code/modules/mafia/map_pieces.dm [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	name = "Mafia Game Board"
+	icon = 'icons/obj/mafia.dmi'
+	icon_state = "board"
+	var/game_id = "mafia"
+
+/obj/mafia_game_board/attack_ghost(mob/user)
+	. = ..()
+	var/datum/mafia_controller/MF = GLOB.mafia_game
+	if(!MF)
+		MF = create_mafia_game()
```
Comment (ghost) [inline code/modules/mafia/map_pieces.dm:26]: how did this compile?

Comment (LetterN) [reply code/modules/mafia/map_pieces.dm:26]: why would it not?

Comment (ghost) [reply code/modules/mafia/map_pieces.dm:26]: nevermind, github hid the diff for controller.dm so I thought it trying to call was an undefined proc

**Comment by JohnFulpWillard** [comment]
god tier

**Pull request merged**